theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -291,6 +291,27 @@ def test_local_gpu_subtensor():
     assert any([isinstance(node.op, cuda.GpuElemwise) for node in topo])
 
 
+def test_local_split():
+    """ Test that the GpuSplit op is being applied and works """
+    # Construct symbolic split
+    x = tensor.vector()
+    splits = tensor.lvector()
```

Comment:
Did you ran the test? It fail for me. I think the problem is that this variable is of dtype long. We only support float32 on the GPU. So make it fvector() and it will probably work.